src/xcode/ENA/ENA/Source/Scenes/RiskLegend/RiskLegendViewController.swift

```diff
@@ -67,74 +84,23 @@ class RiskLegendViewController: DynamicTableViewController {
 		)
 	}
 
-	private var model: DynamicTableViewModel {
+	private func model(maxEncounterAgeInDays: Int) -> DynamicTableViewModel {
```

Comment:
```suggestion
	private func updateModel(maxEncounterAgeInDays: Int) -> DynamicTableViewModel {
```

Reply:
Only an idea :)

Reply:
Hmm, I'm not sure about the name. The function does not update anything, or am I missing something?

Reply:
Perhaps then name it after what is updated or some context? Because model() says really nothing what the func does from my point of view.